### ISSUE TITLE
Termux is on F-Droid?

### ISSUE THREAD

**Issue by angelog0** [termux/termux-app#1499]
I installed Termux with F-Droid and about a month ago I upgraded to 0.92 which is currently installed. Meanwhile I installed other apps with APKPure because I did not find them in F-Droid.

Now APKPure wants to upgrade Termux to 0.94 and searching Termux on F-Droid site shows empty icons, and version says 0.92... The F-Droid app shows a grayed F-Droid icon for Termux...

Is this right?

**Comment by angelog0** [comment]
When I try to upgrade with APKPure, it fails the upgrade...

**Comment by xeffyr** [comment]
Installation source must be same. Do not mix addons/apps from F-Droid, APKPure, Google Play, etc...

**Comment by angelog0** [comment]
> Installation source must be same. Do not mix addons/apps from F-Droid, APKPure, Google Play, etc...

I know this and that was only for testing but you missed the basic question: Is Termux still on F-droid? After 5 day almost all the icons are empty and Termux is at 0.92 (you relesed 0.94, isn't it?)

![Termux_on_FDroid](https://user-images.githubusercontent.com/35931460/77853027-15c4fb80-71e2-11ea-88fb-a15cde5b2362.png)

**Comment by xeffyr** [comment]
> After 5 day almost all the icons are empty and Termux is at 0.92

Because F-Droid doesn't know how to handle vector (in xml) icons.

**Comment by angelog0** [comment]
> > After 5 day almost all the icons are empty and Termux is at 0.92
> 
> Because F-Droid doesn't know how to handle vector (in [xml](https://github.com/termux/termux-app/blob/master/app/src/main/res/drawable/ic_launcher.xml)) icons.

When I installed Termux the first time, it **had** its icon... and I used F-Droid.

Is it to be expected that after 7 days you released 0.94 it still offers 0.92? I am new to F-Droid and Termux and don't know the rate of updates

**Comment by xeffyr** [comment]
Icon problem exists for several months.

Looking through web page source it seems that `https://f-droid.org/repo/icons-640/com.termux.92.xml` is loaded as "image". Obviously, XML format isn't understood as image. This URL also returns 404.

![Screenshot_2020-03-31_18-15-10](https://user-images.githubusercontent.com/25881154/78043379-f366e580-737b-11ea-84c7-56b4ce70868e.png)

> Is it to be expected that after 7 days you released 0.94 it still offers 0.92?

v0.94 metadata was pushed 2 days ago.

F-droid bot has some issues, probably with its infrastructure:
```
== Installed Android Tools ==

INFO: Creating log directory
INFO: Creating temporary directory
INFO: Creating output directory
INFO: Using git version 2.11.0
INFO: Building version 0.94 (94) of com.termux
INFO: Getting source for revision v0.94
INFO: Creating local.properties file at build/com.termux/local.properties
INFO: Creating local.properties file at build/com.termux/app/local.properties
INFO: Cleaned build.gradle of keysigning configs at build/com.termux/app/build.gradle
INFO: Cleaning Gradle project...
ERROR: Could not build app com.termux: Error cleaning com.termux:0.94
==== detail begin ====
Found 6.2.2 via distributionUrl
No hash for gradle version 6.2.2! Exiting...
==== detail end ====
```
